src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslations } from "next-intl";
+
+const Navbar = () => {
+  const t = useTranslations("navbar");
+
+  return (
+    <nav className="dark:bg-gray-900">
+      <ul className="flex p-4 space-x-10">
+        <li>
+          <a
```

Comment:
Since we're using Nextjs, I would recomment to use Link component instead of a tag. Is the good practice here and it renders the things super quick! More info: https://nextjs.org/learn-pages-router/basics/navigate-between-pages/link-component